Tests/PSSwagger.Unit.Tests.ps1

```diff
@@ -118,21 +118,24 @@ Describe "PSSwagger Unit Tests" -Tag @('BVT', 'DRT', 'UnitTest', 'P0') {
         Context 'Get-XDGDirectory Unit Tests (Linux) (Custom)' {
             Mock Get-OperatingSystemInfo -ModuleName PSSwaggerUtility {
                 return @{
-                    IsCore = $true
-                    IsLinux = $true
-                    IsMacOS = $false
+                    IsCore    = $true
+                    IsLinux   = $true
+                    IsMacOS   = $false
                     IsWindows = $false
                 }
             }
 
             Mock Get-EnvironmentVariable -ModuleName PSSwaggerUtility {
                 if ('HOME' -eq $Name) {
                     return '/hometest'
-                } elseif ('XDG_CACHE_HOME' -eq $Name) {
+                }
+                elseif ('XDG_CACHE_HOME' -eq $Name) {
```

Comment:
What did you use to format?

Reply:
Formatting option in VS Code.

---
In reply to: [147846907](https://github.com/PowerShell/PSSwagger/pull/370#discussion_r147846907) [](ancestors = 147846907)